classes/model.php

```diff
@@ -1457,6 +1485,9 @@ protected function update()
 			return false;
 		}
 
+		$this->_update_original();
```

Comment:
Why has this been added? This only causes the original to be updated twice as this already happens in the `save()` method.